source/index.html.md

```diff
@@ -77,9 +77,8 @@ important fields, documented here:
 - `version`: The version of the module. This should conform to 
   [SEMVER](https://semver.org/)
 - `entry`: This is the name of a file that should actually be ran. This key is
-  necessary if, for example, your module registers triggers or commands. An 
-  example of when you wouldn't want to use this field is if your module is a 
-  library, meaning that it is only used by other modules
+  necessary if, for example, your module registers triggers or commands. This field is
+  **REQUIRED** for all modules, without it your module will NOT run.
```

Comment:
Technically, it's only required for modules that wish to run code themselves, libraries do not require it.